build_library/grub_install.sh

```diff
@@ -140,6 +140,10 @@ info "Generating ${GRUB_DIR}/load.cfg"
 ESP_FSID=$(sudo grub-probe -t fs_uuid -d "${LOOP_DEV}p1")
 sudo_clobber "${ESP_DIR}/${GRUB_DIR}/load.cfg" <<EOF
 search.fs_uuid ${ESP_FSID} root \$root
+if [ "$root" = "(memdisk)" ]; then
+    set root="(hd0)"
+    set isoboot="1"
+fi
 set prefix=(memdisk)
```

Comment:
In other cleanup news, prefix is already set to memdisk at this point (implied by calling mkimage with the memdisk option) so this line isn't really needed any more.